lib/ansible/modules/crypto/openssl_pkcs12.py

```diff
@@ -237,7 +237,54 @@ def _check_pkey_passphrase():
         if not state_and_perms:
             return state_and_perms
 
-        return _check_pkey_passphrase
+        if os.path.exists(self.path) and module.params['action'] == 'export':
+            dummy = self.generate(module)
+            self.src = self.path
+            try:
+                pkcs12_privatekey, pkcs12_certificate, pkcs12_other_certificates, pkcs12_friendly_name = self.parse()
```

Comment:
Why not rename `pkcs12_other_certificates` to `pkcs12_ca_certs`?

Reply:
because as above, they don't have to be ca certs, they may be ( and probably usually are ), but pkcs12 is fine with any bundle of certs.